packages/workspace/src/utils/plugins/installed-plugins.ts

```diff
@@ -1,50 +1,27 @@
 import { terminal } from '@angular-devkit/core';
-import { readdirSync } from 'fs';
-import * as path from 'path';
 import { output } from '../output';
 import { CommunityPlugin, CorePlugin, PluginCapabilities } from './models';
 import { getPluginCapabilities } from './plugin-capabilities';
 import { hasElements } from './shared';
 
-function getPackagesFromNodeModules(
-  workspaceRoot: string,
-  requery: boolean = false
-): string[] {
-  let packageList: string[] = [];
-
-  if (!requery && packageList.length > 0) {
-    return packageList;
-  }
-
-  const nodeModulesDir = path.join(workspaceRoot, 'node_modules');
-  readdirSync(nodeModulesDir).forEach((npmPackageOrScope) => {
-    if (npmPackageOrScope.startsWith('@')) {
-      readdirSync(path.join(nodeModulesDir, npmPackageOrScope)).forEach((p) => {
-        packageList.push(`${npmPackageOrScope}/${p}`);
-      });
-    } else {
-      packageList.push(npmPackageOrScope);
-    }
-  });
-
-  return packageList;
-}
-
 export function getInstalledPluginsFromNodeModules(
   workspaceRoot: string,
   corePlugins: CorePlugin[],
   communityPlugins: CommunityPlugin[]
 ): Array<PluginCapabilities> {
-  const corePluginNames = corePlugins.map((p) => p.name);
-  const communityPluginNames = communityPlugins.map((p) => p.name);
-  const packages = getPackagesFromNodeModules(workspaceRoot);
-
-  return packages
-    .filter(
-      (name) =>
-        corePluginNames.indexOf(name) > -1 ||
-        communityPluginNames.indexOf(name) > -1
-    )
+  return [...corePlugins, ...communityPlugins]
```

Comment:
After discussion I'd say it's ok to re-activate this as long as you change the functionality here. Use the `package.json` instead to look for all the installed plugins. This may mean that you have to look through all of the listed packages and check for a `schematics` option in their `package.json`? I can't recall how this function used to work so you'll have to check that.